db/schema.rb

```diff
@@ -48,7 +48,6 @@
     t.string   "description"
     t.datetime "created_at",                    null: false
     t.datetime "updated_at",                    null: false
```

Comment:
The schema you've added here is different as it's missing the boolean "decapper" from "labware_types", but there is no migration in this PR to get rid of it